src/PrestaShopBundle/Form/Admin/Sell/Product/Options/VisibilityType.php

```diff
@@ -65,13 +60,25 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('visibility', ChoiceType::class, [
                 'label' => false,
-                'choices' => $this->productVisibilityChoiceProvider->getChoices(),
+                'choices' => [
+                    $this->trans('Everywhere', 'Admin.Catalog.Feature') => ProductVisibility\Everywhere::VALUE,
+                    $this->trans('Catalog only', 'Admin.Catalog.Feature') => ProductVisibility\Catalog::VALUE,
+                    $this->trans('Search only', 'Admin.Catalog.Feature') => ProductVisibility\Search::VALUE,
+                    $this->trans('Nowhere', 'Admin.Catalog.Feature') => ProductVisibility\Invisible::VALUE,
+                ],
+                'choice_attr' => function (string $value) {
+                    return ['data-description' => isset(self::CHOICES_DESCRIPTIONS[$value]) ?
+                        $this->trans(self::CHOICES_DESCRIPTIONS[$value], 'Admin.Catalog.Feature') : '',
```

Comment:
Worth checking with @jolelievre if this is something that catalog exporter can handle

Reply:
fixed